src/core/amount.js

```diff
@@ -743,8 +762,9 @@ Amount.prototype.to_text = function() {
   // not native
   const offset = this._value.getExponent() - 15;
   const sign = this._value.isNegative() ? '-' : '';
-  const mantissa = utils.getMantissa16FromString(
-    this._value.abs().toString());
+  const mantissa = this._value.isNegative() ?
```

Comment:
Faster. `abs` trigger creation of `BigNumber` object, and `BigNumber` constructor is slowest single part of ther code, so most of my optimizations went into removing unnecessary calls to it constructor.